src/Mobs/ZombiePigman.cpp

```diff
@@ -34,7 +35,11 @@ void cZombiePigman::GetDrops(cItems & a_Drops, cEntity * a_Killer)
 }
 
 
-
+void cZombiePigman::SpawnOn(cClientHandle & a_ClientHandle)
```

Comment:
Style: Please keep the 5 empty lines between top-level functions. It helps readability.